api_tests/base/test_root.py

```diff
@@ -94,12 +95,12 @@ def test_non_admin_scoped_token_does_not_have_admin(self, mock_auth):
             authenticated=True,
             user=self.user._id,
             attributes={
-                'accessToken': token.token_id,
-                'accessTokenScope': [s for s in token.scopes.split(' ')]
+                'accessToken': str(token.token_id),
+                'accessTokenScope': [str(s) for s in token.scopes.split(' ')]
             }
         )
         mock_auth.return_value = self.user, mock_cas_resp
-        res = self.app.get(self.url, headers={'Authorization': 'Bearer {}'.format(token.token_id)})
+        res = self.app.get(self.url, headers={'Authorization': str('Bearer {}'.format(token.token_id))})
```

Comment:
The new version of DRF requires unicode rather than bytestrings, so it was easier to change strings to unicode strings with the import unicode literals. A few things need to be explicitly bytestrings, like the (context?) dicts that are created from the headers here.